### PR TITLE
Newsletters: add 327 (2024-11-01)

### PR DESCRIPTION
**Pull request by harding**
- [x] Lede, releases/RCs, topic entries @harding 
- [x] New topics: duplex channels and DLEQ @harding 
- [x] Illustration for SuperScalar item @harding 
- [x] Notable merges @Gustavojfe

### CODE REVIEW

File: _posts/en/newsletters/2024-11-01-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  the purpose of rebalancing funds.
+
+  Between the root and the leaves are [duplex micropayment
+  channels][topic duplex micropayment channels].  Unlike LN-Penalty
+  channels, duplex channels allow more than two parties to safely share
+  funds; however, they also only allow a relatively small number of
+  state updates compared to LN-Penalty's effectively unlimited number of
+  updates.  The intermediate duplex channels are used to allow
+  rebalances involving the LSP and two end users; these rebalances could
+  securely complete at offchain speeds, allowing user Alice to receive
```
Comment (vostrnad) [inline _posts/en/newsletters/2024-11-01-newsletter.md:60]: Without Bob there's not much need for Alice:
```suggestion
  securely complete at offchain speeds, allowing a user to receive
```

File: _posts/en/newsletters/2024-11-01-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+  in the case of a cooperative close, although it would be less
+  efficient onchain in the case of a unilateral close.
+
+  The proposal received a moderate amount of discussion.  The author
+  said that one of the weaknesses of the proposal is its technical
+  complexity due to the use of multiple different channel types plus the
+  inherent challenge in any channel factory design of managing
+  additional offchain state.  However, it has the advantage of being
+  something that could be implemented by a single team and made
+  interoperable with the standard LN without requiring many changes to
```
Comment (vostrnad) [inline _posts/en/newsletters/2024-11-01-newsletter.md:78]: "Many" or "any"? If "many", what are the changes to the LN protocol that would be required?

File: _posts/en/newsletters/2024-11-01-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+  the LN protocol.
+
+- **Draft BIP for DLEQ proofs:** Andrew Toth [posted][toth dleq] to the
+  Bitcoin-Dev mailing list a draft BIP and a link to an
+  [implementation][dleq imp] for generating and verifying proofs of
+  [discrete log equality][topic dleq] (DLEQ) for the elliptical curve used by
```
Comment (vostrnad) [inline _posts/en/newsletters/2024-11-01-newsletter.md:84]:
```suggestion
  [discrete log equality][topic dleq] (DLEQ) for the elliptic curve used by
```

**Review by bitschmidty — COMMENTED** [review]
So far so good.

I am working to get the SuperScalar Podcast PR open today, targeting merge tomorrow. FYI

File: _posts/en/newsletters/2024-11-01-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+- **Timeout tree channel factories:** ZmnSCPxj [posted][zmnscpxj post1]
+  to Delving Bitcoin and [discussed][deepdive] with Optech contributors a
+  proposal for a new multi-layer [channel factory][topic channel
+  factories] design named _SuperScalar_.  A goal of the design is to
+  provide a construction that could be easily implemented by a single
+  vendor without waiting for large protocol changes that require
+  widespread agreement.  For example, a Lightning service provider (LSP)
+  that distributes wallet software could allow its users to more cheaply
+  open channels and receive inbound liquidity without sacrificing any of
```
Comment (LarryRuane) [inline _posts/en/newsletters/2024-11-01-newsletter.md:22]:
```suggestion
  open channels and receive inbound liquidity without sacrificing
```

File: _posts/en/newsletters/2024-11-01-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+  the users---this places the burden of getting transactions confirmed
+  on the service provider rather than end users.
+
+  The timeout trees used require each involved party to contribute a
+  signature.  This avoids the need for consensus changes but limits the
+  practical maximum number of users in a factory due to the well-known
+  [multi-signer coordination problem][news270 coordination].
+
+  Most of the leaves of the timeout tree are offchain funding
+  transactions for the common type of channel used today (LN-Penalty),
```
Comment (jonatack) [inline _posts/en/newsletters/2024-11-01-newsletter.md:46]: It doesn't look like "LN-Penalty" has been defined anywhere in this repo/website other than with introductory phrases like here, or "In the current LN channel design" in a previous newsletter.

It is also indirectly referred to in the Eltoo topic: "Although eltoo can be used with a penalty mechanism similar to the one used with existing LN channels, eltoo doesn’t need the penalty mechanism in order to be secure."

For anyone not familiar with LN-Penalty, maybe a link to a resource or topic could be good for this term.

**Review by jonatack — COMMENTED** [review]
Looking good and the new topics are 👍

File: _posts/en/newsletters/2024-11-01-newsletter.md (hunk 2 above)
Comment (harding) [reply _posts/en/newsletters/2024-11-01-newsletter.md:78]: Many is correct based on my conversation with ZmnSCPxj in https://bitcoinops.org/en/podcast/2024/10/31/ ; quick rough manual transcription:

49:34 we do need some changes at the protocol layer.  What I'm planning
to propose is that we have some kind of plugable factory concept.  We're
gonig to use an open v1 message to open this channel and then we're
going to have a TLV that says, "hey this is the protocol that we're
using and this is an identifier that specifies the particular protocol
that we're using, and then afterwards you can say "at the factory
layer we did some messaging and we need to change the funding output of
this channel".

**Comment by harding** [comment]
Rebased on master to pull in the deep dive podcast for a link to it.  Made edits (thanks @vostrnad @LarryRuane @jonatack @bitschmidty !), added lede, added topic text\*, added topic entries, removed image fixme (decided not to add one), and updated releases/RCs.  Reviewed @Gustavojfe section (thanks!).

\* There's quite a bit of last-minute text for the DLEQ/PODLE entry.  Sorry.  If anyone leaves a review here post-merge, I'll prioritize addressing that feedback Friday.

**Review by bitschmidty — APPROVED** [review]
Reviewed topics, lede, updates. Fixed up a couple minor items. Looks good!

Thank you @harding @Gustavojfe @LarryRuane @jonatack @vostrnad

**Pull request merged**